rio/src/RIO.hs

```diff
@@ -1,5 +1,6 @@
 module RIO
   ( module RIO.Prelude.Display
+  , module RIO.Prelude.Exit
```

Comment:
Please include a mention in the `ChangeLog.md` about the addition.

Reply:
This would end up in a version bump to `0.1.9.1`? or does the new entry go under `0.1.9.0`? or does this go under a "next release" heading in the `ChangeLog`?

Reply:
Since I haven't released 0.1.9.0 yet, it should be part of that release.

Reply:
Also, this new module should be added to the hidden module list in `package.yaml`

Reply:
I'm not sure about the hidden module list, but I'm assuming you mean the existing `other-modules`. I've updated that, thanks!